clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -178,6 +178,15 @@ public CommitRequestManager(
      */
     @Override
     public NetworkClientDelegate.PollResult poll(final long currentTimeMs) {
+        if (coordinatorRequestManager.coordinator().isEmpty() && closing) {
+            CommitFailedException exception = new CommitFailedException(
```

Comment:
Should we consider propagating the coordinator error that is causing the empty coordinator?